BUILDING.md

```diff
@@ -3,7 +3,7 @@ Building JanusGraph
 
 Required:
 
-* Java 7 (0.5 and earlier) or Java 8 (0.9 and later)
+* Java 8 (0.9 and later)
```

Comment:
No need to specify 0.9 and later as that referred to Titan versions. I suggest specifying Maven 3, but I don't think we're picky on specific dot version on that.